tests/phpunit/test-cache.php

```diff
@@ -114,10 +143,15 @@ public function test_redis_reload_force_cache_flush() {
 		// Force a bad connection
 		$redis_server['host'] = '127.0.0.1';
 		$redis_server['port'] = 9999;
-		$this->cache->redis->connect( $redis_server['host'], $redis_server['port'], 1, null, 100 );
+		$this->cache->build_client_connection( $this->cache->build_client_parameters( $redis_server ) );
```

Comment:
I replaced `redis->connect` with `build_client_connection` because Predis' `connect()` method doesn't behave the same way PHPRedis' does. Predis only takes the connection details in it's Client class' constructor and `connect` simply creates a connection manually.